docs/guides/postman-collection.md

```diff
@@ -0,0 +1,87 @@
+# Postman Collections support
+
+Prism offers a _limited_ support for Postman Collection. The basic workflow of using Prism (both from the CLI and deployed in a Docker container) are fundamentally the same, however there are some known limitations that it's important to keep in mind:
+
+## Authentication
+
+Postman collections support more authentication methods than OpenAPI 2 and 3, although some of these are a specialisation of another method included in their support; this is mostly to drive the UI of their application.
+
+In case the authentication method does not have a corresponding element in the OpenAPI specification, it can ultimately be expressed as a header. In such a case, Prism will "downgrade" according to the following table:
+
+| Postman Collection | Transformation         |
+| ------------------ | ---------------------- |
+| apikey             | apikey                 |
+| bearer             | http + bearer scheme   |
+| digest             | http + digest scheme   |
+| oauth1             | http + bearer scheme   |
+| oauth2             | http + bearer scheme   |
+| awk                | Hardcoded header value |
+| aws                | Hardcoded header value |
+| akamai             | Hardcoded header value |
+| ntlm               | Hardcoded header value |
+
+## Events
+
+Events are linked to the Postman hosted platform, and so Prism is going to skip this section entirely.
+
+**Warning:** these scripts/event handlers could potentially modify variable values, with the consequence of changing the way the subsequent requests would work.
+
+## Request Body
```

Comment:
We don't advertise GraphQL Support anywhere in Prism so I dont think anyone would be surprised that its not working for Postman? 

It's also clear that a disabled flag means its ignored by Prism that's a reasonable default. We can remove this too.

Reply:
Ok fair, I've removed the section